src/globemap.ts

```diff
@@ -222,7 +222,7 @@ module powerbi.extensibility.visual {
         private mapTextures: THREE.Texture[];
         public barsGroup: THREE.Object3D;
         private readyToRender: boolean;
-        private deferredRenderTimerId: number;
+        private deferredRenderTimerId: NodeJS.Timer;
```

Comment:
It can not be defined as NodeJS.Timer as this file is executed in browser context not node.js.
It must be number.